### ISSUE TITLE
remove logging controller class (Trac #23)

### ISSUE THREAD

**Issue by cfobel** [wheeler-microfluidics/microdrop#27]
Remove logging controller class, since main_window_controller can implement ILoggingPlugin to handle error/warning plugin signals directly.

Migrated from http://microfluidics.utoronto.ca/ticket/23
```json
{
    "status": "closed", 
    "changetime": "2014-04-17T19:39:01", 
    "description": "Remove logging controller class, since main_window_controller can implement ILoggingPlugin to handle error/warning plugin signals directly.", 
    "reporter": "cfobel", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1397763541728826", 
    "component": "microdrop", 
    "summary": "remove logging controller class", 
    "priority": "major", 
    "keywords": "", 
    "version": "0.1", 
    "time": "2012-01-04T03:25:18", 
    "milestone": "Microdrop 1.0", 
    "owner": "cfobel", 
    "type": "defect"
}
```

**Comment by cfobel** [comment]
Trac update at 20120104T04:50:22: 
* christian@fobel.net commented:

> commit 8a0ea9eda4e2885d0244f4d92f0593e87c9451bf
> Author: Christian Fobel <christian@fobel.net>
> Date:   Tue Jan 3 23:43:31 2012 -0500
> 
>     Remove logging controller class...
>     
>     Remove logging controller class, since main_window_controller can implement
>     ILoggingPlugin to handle error/warning plugin signals directly. Fixes #27.


* christian@fobel.net changed resolution from "" to "fixed"
* christian@fobel.net changed status from "new" to "closed"

**Comment by cfobel** [comment]
Trac update at 20140417T19:39:01: ryan changed component from "core" to "microdrop"
